go.mod

```diff
@@ -37,17 +36,16 @@ require (
 	github.com/smartystreets/goconvey v0.0.0-20180222194500-ef6db91d284a // indirect
 	github.com/stretchr/testify v1.3.0 // indirect
 	github.com/tidwall/pretty v0.0.0-20180105212114-65a9db5fad51 // indirect
+	github.com/xanzy/go-gitlab v0.15.0
```

Comment:
Resolve conflicts and regenerate using `go mod tidy`

Reply:
I fixed it.